### PR TITLE
Fixed links in example to be relative to current URL

### PR DESCRIPTION
**Pull request by astef**

### CODE REVIEW

File: examples/static-site/src/index.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ layout: layout.html
 
 <h2>Read what I have to say</h2>
 
-<a href="/posts/first-post/">First post</a>
+<a href="./posts/first-post/">First post</a>
```
Comment (woodyrew) [inline examples/static-site/src/index.md:7]: Relative links would be better expressed as
```
<a href="posts/first-post/">First post</a>
```

**Review by woodyrew — COMMENTED** [review]
Thanks for the PR. Are the site based links causing an issue?
Relative links can become difficult to work with if you're re-using content throughout a site.

**Comment by astef** [comment]
I'm usually deploy the site to `http://localhost/siteName/...`, rather than top-level site with random port.

Also, my use case for the metalsmith is to generate static documentation for the existing web-site. So it is important for me to make sure it will be working on any nesting level.

And you're right about difficulties with re-using content throughout a site, Home links are still "broken" in the example, and the fix is not so trivial for them.

**Comment by webketje** [comment]
@astef the better solution would be to have en ENV var defined in a `.env` file or set on the server, then pass to metalsmith metadata as `BASE_URL` and set the links to `<a href="{{ BASE_URL }}/path/to/post">...</a>`. Closing as this PR is no longer relevant

**Pull request closed without merging**